core/transport/src/main/java/alluxio/grpc/PMode.java

```diff
@@ -674,6 +674,5 @@ public PMode parsePartialFrom(
   public alluxio.grpc.PMode getDefaultInstanceForType() {
     return DEFAULT_INSTANCE;
   }
-
```

Comment:
can you revert the change to this file (the added empty line)?

Reply:
Thanks. reverted.